docs/developers_guide/framework/config.md

```diff
@@ -40,17 +40,16 @@ cases in a configuration, as in this example from
 {py:func}`setup.setup_case()`:
 
 ```python
-# add the config options for the test group (if defined)
-test_group = test_case.test_group.name
-config.add_from_package(f'polaris.{component}.tests.{test_group}',
-                        f'{test_group}.cfg', exception=False)
+# add the config options for the task (if defined)
+config.add_from_package(task.__module__,
+                        f'{task.name}.cfg', exception=False)
```

Comment:
This doesn't seem to exist anymore. Would you like to use an example from one of the test categories, e.g. https://github.com/E3SM-Project/polaris/blob/718dbb49b95d1c12e56462c7bfd7002ded5a4db8/polaris/ocean/tasks/inertial_gravity_wave/convergence/__init__.py#L36-L42

Reply:
That suggestion would be fine, too!  But I do think this line exists here:
https://github.com/xylar/polaris/blob/remove-test-groups/polaris/setup.py#L178-L180

Reply:
Ok. I didn't see that, so just change `{py:func} setup.setup_case():` to `{py:func} setup.setup_task():`

Reply:
Done, thanks!